crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -520,13 +520,13 @@ impl Space {
         ep_hnd: Tx2EpHnd<wire::Wire>,
         config: Arc<KitsuneP2pConfig>,
     ) -> Self {
-        let gossip_mod_fact = if &config.tuning_params.gossip_strategy == "simple-bloom" {
-            crate::gossip::simple_bloom::factory()
-        } else {
-            panic!(
+        let gossip_mod_fact = match config.tuning_params.gossip_strategy.as_str() {
+            "simple-bloom" => crate::gossip::simple_bloom::factory(),
+            "sharded-gossip" => crate::gossip::sharded_gossip::factory(),
```

Comment:
Not a big deal and not relevant but I wonder why this isn't an enum

Reply:
Because all the tuning params are strings. I don't think we can set an enum?